src/main/java/mate/academy/spring/model/dto/response/ShoppingCartResponseDto.java

```diff
@@ -0,0 +1,33 @@
+package mate.academy.spring.model.dto.response;
+
+import java.util.List;
+
+public class ShoppingCartResponseDto {
+    private Long id;
+    private List<Long> ticketsIds;
+    private Long usersId;
```

Comment:
[When choosing the fields to be returned in your ShoppingCartResponseDto, keep in mind that cartId and userId hold the same value.](https://mate-academy.github.io/jv-program-common-mistakes/java-spring/rest-practice/jv-spring-rest-practice_checklist#:~:text=When%20choosing%20the%20fields%20to%20be%20returned%20in%20your%20ShoppingCartResponseDto%2C%20keep%20in%20mind%20that%20cartId%20and%20userId%20hold%20the%20same%20value.)